pos_accented_search/__openerp__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+{
+    'name': "pos_accented_search",
```

Comment:
this should be a user friendly name, the technical name is already the module name